library/src/level2/rocsparse_gebsrmv.cpp

```diff
@@ -397,6 +397,9 @@ rocsparse_status rocsparse::gebsrmv_template(rocsparse_handle          handle, /
     // row_block_dim == 1 and col_block_dim == 1 is the CSR case
     if(row_block_dim == 1 && col_block_dim == 1)
     {
+        static constexpr bool fallback_algorithm = true;
+        static constexpr bool no_force_conj      = false;
```

Comment:
Should this be:

static constexpr bool force_conj = false;

Reply:
That was on purpose. I found it more explicit when passing the parameter to the routine.